README.md

```diff
@@ -36,8 +36,6 @@ For example: the following configuration would use the `ssd` framework:
 }
 ```
 
-Note: if you are going to use *get_detections_from_camera()*, you will need to set any camera that you plan to use as a dependency for this service.
```

Comment:
Why is this removed?  Isn't it still true?

Reply:
It repeated a line before this, so you say it twice